### ISSUE TITLE
Don't report unreported comments on deleted posts

### ISSUE THREAD

**Issue by LunarWatcher** [SOBotics/QueenReporter#8]
If the parent post of a comment is deleted, Queen can't fetch the comment. If the comment has been reported, giving feedback is obviously not a problem. But if it isn't reported, sending a report on the comment is pointless because Queen can't fetch the comment anyways. 

There's a specific HTML class present for all deleted posts: `deleted-answer`. 

If you get the UID of the parent post (if it's on the question, it's considerably easier, because you have the `question` ID), you can check whether `#answer-<question UID without brackets>` has a class called `deleted-answer`. For the question, `#question.deleted-answer`. 

If the question is deleted, so are the answers, so there's no need to check for both answer and question deletion. If the question is deleted and the answer isn't, that's an issue triggered by flag clearing and is usually repaired. The chances of this happening is virtually 0, and therefore isn't worth paying much attention to. 

TL;DR:

If comment.getParentPost().hasClass("deleted-answer") don't report unreported comment, but give feedback on already reported comment
